pkgs/development/python-modules/enochecker_core/default.nix

```
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "enochecker_core";
```

```suggestion
  pname = "enochecker-core";
```
`pname` should be normalized according to [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names).
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines

---

With this change, `fetchPypi` fails since it now tries to retrieve `enochecker-test-0.9.0.tar.gz` instead of `enochecker_test-0.9.0.tar.gz`. How about the following?

```
  pname = "enochecker-test";
  # ...
  src = fetchPypi {
    inherit version;
    pname = "enochecker_test";
    hash = "sha256-M0RTstFePU7O51YVEncVDuuR6F7R8mfdKbO0j7k/o8Q=";
  };

```

---

done.